### ISSUE TITLE
The at() matcher has been deprecated.

### ISSUE THREAD

**Issue by mauriciofauth** [phpmyadmin/phpmyadmin#16332]
The `PHPUnit\Framework\TestCase::at` method has been deprecated.

> The at() matcher has been deprecated. It will be removed in PHPUnit 10. Please refactor your test to not rely on the order in which methods are invoked.

**Comment by elaynelemos** [comment]
Hey! I would like to work on it. May I?

**Comment by ibennetch** [comment]
Sure Elayne, that would be great! Let us know if you need any help.

**Comment by williamdes** [comment]
- [ ] PhpMyAdmin\Tests\Controllers\Database\StructureControllerTest::testSynchronizeFavoriteTables - The at() matcher has been deprecated.
- [ ] PhpMyAdmin\Tests\Controllers\Server\PluginsControllerTest::testIndex - The at() matcher has been deprecated.
- [ ] PhpMyAdmin\Tests\Database\CentralColumnsTest::testDeleteColumnsFromList - The at() matcher has been deprecated.

- [ ] PhpMyAdmin\Tests\InsertEditTest
```
7) PhpMyAdmin\Tests\InsertEditTest::testShowEmptyResultMessageOrSetUniqueCondition - The at() matcher has been deprecated.
8) PhpMyAdmin\Tests\InsertEditTest::testLoadFirstRow - The at() matcher has been deprecated.
9) PhpMyAdmin\Tests\InsertEditTest::testSetSessionForEditNext - The at() matcher has been deprecated.
10) PhpMyAdmin\Tests\InsertEditTest::testExecuteSqlQuery - The at() matcher has been deprecated.
11) PhpMyAdmin\Tests\InsertEditTest::testExecuteSqlQueryWithTryQuery - The at() matcher has been deprecated.
12) PhpMyAdmin\Tests\InsertEditTest::testGetCurrentValueForDifferentTypes - The at() matcher has been deprecated.
13) PhpMyAdmin\Tests\InsertEditTest::testVerifyWhetherValueCanBeTruncatedAndAppendExtraData - The at() matcher has been deprecated.
14) PhpMyAdmin\Tests\InsertEditTest::testGetTableColumns - The at() matcher has been deprecated.
```

- [ ] PhpMyAdmin\Tests\Plugins\Auth\AuthenticationCookieTest
```
15) PhpMyAdmin\Tests\Plugins\Auth\AuthenticationCookieTest::testAuthCheckDecryptPassword - The at() matcher has been deprecated.
16) PhpMyAdmin\Tests\Plugins\Auth\AuthenticationCookieTest::testAuthFailsDBI - The at() matcher has been deprecated.
17) PhpMyAdmin\Tests\Plugins\Auth\AuthenticationCookieTest::testAuthFailsErrno - The at() matcher has been deprecated.

```

- [ ] PhpMyAdmin\Tests\Plugins\Auth\AuthenticationHttpTest::testAuthFails - The at() matcher has been deprecated.

- [ ] PhpMyAdmin\Tests\Plugins\Auth\AuthenticationSignonTest
```
19) PhpMyAdmin\Tests\Plugins\Auth\AuthenticationSignonTest::testAuthFailsMySQLError - The at() matcher has been deprecated.
20) PhpMyAdmin\Tests\Plugins\Auth\AuthenticationSignonTest::testAuthFailsConnect - The at() matcher has been deprecated.
```
- [ ] PhpMyAdmin\Tests\Plugins\Schema\DiaRelationSchemaTest::testSetProperty - The at() matcher has been deprecated.

- [ ] PhpMyAdmin\Tests\Plugins\Schema\EpsRelationSchemaTest
```
22) PhpMyAdmin\Tests\Plugins\Schema\EpsRelationSchemaTest::testConstructor - The at() matcher has been deprecated.
23) PhpMyAdmin\Tests\Plugins\Schema\EpsRelationSchemaTest::testSetPageNumber - The at() matcher has been deprecated.
```
- [ ] PhpMyAdmin\Tests\Plugins\Schema\PdfRelationSchemaTest::testConstructor - The at() matcher has been deprecated.
- [ ] PhpMyAdmin\Tests\Plugins\Schema\SvgRelationSchemaTest::testConstructor - The at() matcher has been deprecated.
- [ ] PhpMyAdmin\Tests\Server\PrivilegesTest::testGetHtmlToDisplayPrivilegesTable - The at() matcher has been deprecated.

- [ ] PhpMyAdmin\Tests\Server\UserGroupsTest
```
27) PhpMyAdmin\Tests\Server\UserGroupsTest::testGetHtmlForUserGroupsTableWithUserGroups - The at() matcher has been deprecated.
28) PhpMyAdmin\Tests\Server\UserGroupsTest::testDeleteUserGroup - The at() matcher has been deprecated.
```

**Comment by mauriciofauth** [comment]
Closed by a7d6df17d604a8056de4c127f6c9296d34755a95.
